### PR TITLE
Add HP

### PR DESCRIPTION
**Pull request by pitdicker**
The category 'retail' doesn't feel like a great fit for HP, but it is the same as for Dell.
Their documentation for 2FA almost immediately redirects to a login page :-(.

### CODE REVIEW

File: img/h/hp.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 36 36" xmlns="http://www.w3.org/2000/svg"><path d="m18 36c-0.179 0-0.3571-0.0033-0.5349-0.0084l3.678-10.11h5.062c0.8886 0 1.864-0.6831 2.168-1.518l3.991-10.96c0.6529-1.795-0.3751-3.263-2.285-3.263h-7.023l-5.893 16.19h-0.0023l-3.341 9.18c-7.926-1.884-13.82-9.009-13.82-17.51 0-8.257 5.56-15.21 13.14-17.34l-3.447 9.474h-9.8e-4l-5.731 15.74 3.815 4e-4 4.873-13.38h2.867l-4.874 13.38 3.813 6e-4 4.542-12.48c0.6533-1.795-0.3748-3.263-2.283-3.263h-3.207l3.682-10.12c0.2684-0.01188 0.5378-0.01892 0.809-0.01892 9.941 0 18 8.059 18 18 0 9.941-8.059 18-18 18zm10.89-23.51h-2.866l-4.014 11.01h2.866l4.014-11.01z" fill="#0096D6"/></svg>
```
Comment (phallobst) [inline img/h/hp.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 36 36"><path fill="#0096D6" d="M18 36h-.5L21 25.9h5.1c.9 0 1.9-.7 2.2-1.5l4-11c.6-1.8-.4-3.3-2.3-3.3h-7l-6 16.2-3.3 9.2A18 18 0 0 1 13.1.7L9.7 10 4 26h3.8l4.8-13.4h3l-5 13.4h3.9L19 13.4c.7-1.8-.4-3.3-2.3-3.3h-3.2l3.7-10L18 0a18 18 0 1 1 0 36Zm10.9-23.5H26l-4 11H25l4-11Z"/></svg>
```

File: entries/h/hp.com.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "HP": {
+    "domain": "hp.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://account.id.hp.com/webcomponents/static/mfa.html",
```
Comment (phallobst) [inline entries/h/hp.com.json]: That link forwards to the login page after just one second (at least for me), so it's probably not useful to be used as a public documentation page. It does explain the TOTP setup process, however.
```suggestion
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: img/h/hp.com.svg (hunk 1 above)
Comment (pitdicker) [reply img/h/hp.com.svg]: I ran the SVG through Inkscape with the save as optimized SVG. Is this the result of another tool? I notice it loses most of the decimals. (Not that I care all that much about the logo to be honest).

Comment (phallobst) [reply img/h/hp.com.svg]: You can play around with [SVGOMG](https://jakearchibald.github.io/svgomg/) and try the "Precision" slider. Usually 1 or 2 is sufficient and will show no distortion, even on full zoom.

Comment (pitdicker) [reply img/h/hp.com.svg]: Thank you, sorry for the slow reply.

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**